src/pages/lesson-details/LessonDetails.tsx

```diff
@@ -80,6 +90,11 @@ const LessonDetails = () => {
 
   return (
     <PageWrapper>
+      <Box sx={styles.btnContainer}>
```

Comment:
I guess `Box` is useless. You can add to the button style `margin-left: auto` and your button move to the right

Reply:
done